src/main/resources/config.sample.yml

```diff
@@ -45,30 +48,45 @@ platforms :
       url: http://example.com/
 
     - type: pybossa
+
       # the key to access the pybossa api
       apiKey: 8ec92fa1-1bd1-42ad-8524-3d2bab4588b1
+
       # the url to the pybossa api
       url: http://localhost:5000/api
+
+      # name which is used to display
       name: Pybossa
+
       # the id of the pybossa project to use for all experiments
       projectId: 1
+
       # if calibrations are allowed on this platform
       calibrationsAllowed: true
 
 # basic configurations for the mail-module
-mail :
+mail:
```

Comment:
1. do you really want to tell me that a user is impressed if the style is unified ? I guess the administrator will be proud if you managed to configure the software and understand the difference between workerLocalUrl and workerPublicUrl or the fact that notificationsEmail is not used by notifications :>

2. I said its _NOT_ about this line.

3. Yes it should be consistent, is it now: NO, is it after this patch: NO.